lib-android-common/src/main/java/com/devinci/lib/util/Versions.java

```diff
@@ -0,0 +1,38 @@
+package com.devinci.lib.util;
+
+import android.os.Build;
+
+/**
+ * Utility methods to check Android version.
+ */
+public enum Versions {
+  ;
```

Comment:
I have mixed feelings with this util class approach - auto-completion suggests enum methods :/